src/src/experiment-tracking/components/App.tsx

```diff
@@ -56,9 +57,28 @@ const classNames = {
 const InteractionTracker = ({ children }: any) => children;
 
 class App extends Component {
-  state = {
-    version: 'unknown',
+  state: {
+    selectedNamespace: string; namespaces: string[];
+    version: string;
   };
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      selectedNamespace: 'ns1',
```

Comment:
why `selectedNamesapce` in state initially -> `ns1`? debug?

Reply:
Ops, sorry I'll fix it